packages/salesforcedx-vscode-soql/package.json

```diff
@@ -19,7 +19,9 @@
     "Programming Languages"
   ],
   "dependencies": {
-    "@salesforce/soql-builder-ui": "0.0.2"
+    "@salesforce/soql-builder-ui": "0.0.2",
+    "@salesforce/soql-language-server": "0.1.0",
```

Comment:
This package is currently only published to an internal Salesforce npm registry, because it depends on another internal Salesforce package. This PR should not be merged until those packages are made public.

Reply:
@jgellin-sf We should not depend on soql-parser being made public. Can we still publish the soql-language-server to public npm registry? I think we can use bundleDependencies.

Reply:
I'll give it a try.

Reply:
Ok that worked. See the 'Bump version of language server for bundled parser' commit.

Reply:
The bundling change is merged and the bundled package is available on npmjs. This issue should no longer hold up merging this PR.